### ISSUE TITLE
0:// in exclude attachments

### ISSUE THREAD

**Issue by eriktelepovsky** [CreativeDream/jQuery.filer#168]
Hi,

I just want to ask, what is the purpose of having "0://" prefix in the list of excluded attachments value?

```
'jfiler-items-exclude-attachments-0': ['["0://IMG_6388_JaO2vEi.PNG","0://IMG_6387_fhz7vSF.PNG"]'],
```

**Comment by CreativeDream** [comment]
https://github.com/innostudio/fileuploader

**Comment by eriktelepovsky** [comment]
I am sorry, but I don't understand your answer. Can you explain, please?
